week-3/Homework/mandatory/3-space-colonies.js

```diff
@@ -8,7 +8,9 @@
   NOTE: don't include any element that is not a "family".
 */
 
-function colonisers() {
+function colonisers(families) {
+
+  return families.filter(familie => familie.includes("family")).filter(familie => familie.startsWith("A"))
```

Comment:
A little tip for formatting your code to make it easier to read *chained* methods:

```js
return families
  .filter((familie) => familie.includes("family"))
  .filter((familie) => familie.startsWith("A"));
```